deploy-disconnected-registry/common.sh

```diff
@@ -207,7 +207,7 @@ elif [[ ${1} == "edgecluster" ]]; then
           timeout=0
           ready=false
           echo "DEBUG: oc --kubeconfig=${EDGE_KUBECONFIG} get route -n ${REGISTRY} ${REGISTRY}-quay -o jsonpath={'.status.ingress[0].host'}"
-          while [ "$timeout" -lt "100" ]; do
+          while [ "$timeout" -lt "1000" ]; do
```

Comment:
Should we lower this? 1000 sounds like too much

Reply:
I'm thinking about physical servers...

Reply:
only if route is not present...

Reply:
Yes, but 1000 is a 1h sleep. If the server hasn't come back online after 10/15 mins then I'd say we have a bigger problem.

Reply:
200 ???